app/src/org/commcare/activities/FormRecordListActivity.java

```diff
@@ -139,8 +139,8 @@ public String getMessage() {
     }
 
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+    protected void onCreateSessionSafe(Bundle savedInstanceState) {
```

Comment:
Should fix https://fabric.io/dimagi/android/apps/org.commcare.dalvik/issues/59f979c161b02d480d8ff90d?time=last-thirty-days